src/convertToObject.js

```diff
@@ -1,12 +1,19 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((line) => line.trim())
+    .reduce((acc, line) => {
+      const [property, value] = line.split(':');
+
+      if (property && value) {
+        const trimmedProperty = property.trim();
+        const trimmedValue = value.trim();
+
+        acc[trimmedProperty] = trimmedValue;
+      }
+
+      return acc;
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. You could format your code as follows:

javascript
return sourceString
  .split(';')
  .filter((line) => line.trim())
  .reduce((acc, line) => {
    const [property, value] = line.split(':');
    if (property && value) {
      const trimmedProperty = property.trim();
      const trimmedValue = value.trim();
      acc[trimmedProperty] = trimmedValue;
    }
    return acc;
  }, {});
```
This way, it's easier to understand the flow of your operations.